app/frontend/src/components/agriseafoodopscreening/admin/SubmissionsTable.vue

```diff
@@ -48,7 +48,7 @@
 
 <script>
 import GeneratePdfButton from '@/components/common/GeneratePdfButton.vue';
-import minesOperatorScreeningService from '@/services/minesOperatorScreeningService';
+import agriSeafoodOpScreeningService from '@/services/agriSeafoodOpScreeningService';
 
 export default {
   name: 'MinesSubmissionsTable',
```

Comment:
i think this needs changing at some point

Reply:
Fixed